modules/apps/depot/depot-service/src/main/java/com/liferay/depot/service/impl/DepotEntryLocalServiceImpl.java

```diff
@@ -155,21 +155,29 @@ public DepotEntry addDepotEntry(
 	}
 
 	@Override
-	public DepotEntry deleteDepotEntry(long depotEntryId)
+	public DepotEntry deleteDepotEntry(DepotEntry depotEntry)
 		throws PortalException {
 
-		DepotEntry depotEntry = depotEntryPersistence.fetchByPrimaryKey(
-			depotEntryId);
-
 		if (_isStaged(depotEntry)) {
 			throw new DepotEntryStagedException(
-				"Unstage depot entry " + depotEntryId + " before deleting it");
+				"Unstage depot entry " + depotEntry.getDepotEntryId() +
+					" before deleting it");
 		}
 
 		_resourceLocalService.deleteResource(
 			depotEntry, ResourceConstants.SCOPE_INDIVIDUAL);
 
-		return super.deleteDepotEntry(depotEntryId);
+		return super.deleteDepotEntry(depotEntry);
+	}
+
+	@Override
+	public DepotEntry deleteDepotEntry(long depotEntryId)
+		throws PortalException {
+
+		DepotEntry depotEntry = depotEntryPersistence.fetchByPrimaryKey(
```

Comment:
`depotEntry` is used only once; can you inline it?

Reply:
Hi @adolfopa 

I have updated the commit with the requested change, see: https://github.com/liferay-lima/liferay-portal/pull/3417/commits/dfbdb247fd0487c59d2577b97a147c64e4e1abe9#diff-356ff59c01041287ff92ac5c64090d41c29c0ef8666cf52fe6022721e43f828dR177